### DISCUSSION TITLE
Hello everyone [Aug 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6297] [Discord Threads]
> 🙋‍♂️ **microbica**  
#### `Mysql as DB`  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1131047456327618620/image.png)  
![image1](https://cdn.discordapp.com/attachments/661905455894888493/1131047456621203527/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **microbica**  
#### My package.json   
####    
#### ```{   
####   "name": "help",   
####   "version": "1.0.0",   
####   "description": "",   
####   "main": "index.js",   
####   "scripts": {   
####     "test": "echo \"Error: no test specified\" && exit 1",   
####     "start": "node index.js"   
####   },   
####   "keywords": [],   
####   "author": "",   
####   "license": "ISC",   
####   "dependencies": {   
####     "dotenv": "^8.2.0",   
####     "express": "^4.17.1",   
####     "forever-monitor": "^3.0.3",   
####     "noco": "^0.2.0",   
####     "nocodb": "^0.109.4"   
####   }   
#### }```  
#  
> 💁 **o1lab**  
#### send the error as DM to <@682854952657616902>  
#  
> 🙋‍♂️ **microbica**  
#### No answer 😦 <@310157751395287040>  
#  
> 👨‍💻 **pranavxc**  
#### Use `npm run watch:run`   
####    
#### Ref: https://docs.nocodb.com/engineering/development-setup/#build-backend  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1131047456910618624)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
